provisioning/device/src/ProvisioningClientOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client
+{
+    /// <summary>
+    /// Options that allow configuration of the Provisioning device client instance during initialization.
+    /// </summary>
+    public class ProvisioningClientOptions
+    {
+        /// <summary>
+        /// Creates an instances of this class with the default transport settings.
+        /// </summary>
+        public ProvisioningClientOptions()
+        {
+            ProvisioningTransportSettings = new ProvisioningTransportHandlerAmqp();
```

Comment:
As I just changed the hub device client default to MQTT, can we change the default here too?

Reply:
I think it is more important to be inline with the protocol the hub client uses, considering some customers will need to open up firewall ports and such.

We know MQTT isn't as rich as AMQP, even in hub. The whole team is pushing to improve the MQTT experience, so I think we just need to keep pushing it.